gluoncv/utils/parallel.py

```diff
@@ -93,12 +93,13 @@ def __init__(self, module, ctx_list=None, sync=False):
     def __call__(self, inputs, *targets, **kwargs):
         # the inputs should be the outputs of DataParallelModel
         if not self.ctx_list:
-            return self.module(inputs, *targets, **kwargs)
+            return self.module(*inputs, *targets, **kwargs)
+        inputs, _ = split_load_kwargs(inputs, kwargs, self.ctx_list)
```

Comment:
The inputs should be the output of DataParallelModel, which is already split loaded. What's the motivation for this change?

Reply:
I have only one GPU,  when used with aux=True in FCN,  the outputs is a tuple, and the length of the outputs is 2, so cause the assert error, because the len(self.ctx_list) is 1. If used with aux=False in FCN, the length of the outputs is the batch_size, e.g. 4, and also cause assert error.

Reply:
Get you. I will take into this. Thx